### PR TITLE
Add FileChanges.new_created_files (part of fix for bsc#1027582)

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

File: library/system/src/modules/FileChanges.rb [hunk 1]
```diff
@@ -213,16 +213,24 @@ def CheckFiles(files)
       ret == :ok
     end
 
+    # Files that are really new
+    #
+    # @param files [Array<String>] candidate files that may be new
+    # @return [Array<String>]
+    def new_created_files(files)
```
Comment (jreidinger) [inline library/system/src/modules/FileChanges.rb]: name is a bit misleading for me, maybe `newly_created_files`?

Comment (jreidinger) [inline library/system/src/modules/FileChanges.rb]: current name looks like it create new object of type CreatedFiles.

Comment (ancorgs) [inline library/system/src/modules/FileChanges.rb]: It's for consistency with `CheckNewCreatedFiles` (the new method is extracting part of its logic).

Comment (jreidinger) [inline library/system/src/modules/FileChanges.rb]: hmm, it is different naming convention and also in combination of CheckNew sounds for me a bit different. I actually notice it first time in second pull request, where it hit my eyes, but I let it up to you

**Pull request merged**